### PR TITLE
💎(site) Update prohibited strings

### LINKED ISSUE #298: Update prohibited strings
There is now a finalized list of inclusive language (internal link, sorry) that we should align our prohibited strings linting to

### PR DESCRIPTION
**Pull request by nohe427**
This updates the list of prohibited strings to be a close match to the
list that is maintained internally.

- Resolves #298

### CODE REVIEW

File: .remarkrc.js [hunk 1]
```diff
@@ -40,15 +51,37 @@ exports.plugins = [
...
+    { no: 'idiot' },
+    { no: 'mongoloid' },
+    { no: 'permenant(-|\s)?child' },
+    { no: 'short(-|\s)?bus' },
+    { no: 'slow(-|\s)?learner' },
+    { no: 'daft' },
+    { no: 'imbecil' },
+    { no: 'stupid' },
+    { no: 'moronic' },
+    { no: '(-|\s)?tard' }
```
Comment (Snugug) [inline .remarkrc.js:84]:
```suggestion
    { no: '(-|\s)?tard(ed)?' }
```
Adding this at the end should cover everything that ends in `tard` or `tarded`, and let us get rid of stand-alone `retarded`

File: .remarkrc.js [hunk 2]
```diff
@@ -25,13 +25,24 @@ exports.plugins = [
...
+    { no: 'blacklist' },
+    { no: 'whitelist' },
+    { no: 'graylist' },
+    { no: 'dark(-|\s)?greylist' },
+    { no: 'light(-|\s)?greylist' },
+    { no: 'tribal(-|\s)?knowledge' },
+    { no: 'tribal(-|\s)?wisdom' },
     { no: 'tribal' },
     { no: 'tribalism' },
+    { no: 'tribal(-|\s)?division'},
```
Comment (Snugug) [inline .remarkrc.js:37]:
```suggestion
    { no: 'tribal(-|\s|ism)?(knowledge|wisdom|division)?'},
```

Can this and line 33 and 34 be combined, like we do on line 68?

File: .remarkrc.js [hunk 3]
```diff
@@ -25,13 +25,24 @@ exports.plugins = [
...
     { no: 'nitty(-|\s)?gritty' },
     { no: 'voodoo' },
     { no: 'off(-|\s)?the(-|\s)?reservation' },
+    { no: 'the(-|\s)?natives'},
```
Comment (Snugug) [inline .remarkrc.js:41]: Can line 42 be updated to include both with and without the the? Something like `(the(-|\s)?)?natives`?

**Review by Snugug — CHANGES_REQUESTED** [review]
Thanks for working on this!

There are a few places where I think additions you've made are already covered, and where things can be combined. If you could take a look that'd be appreciated!

Thanks!

**Comment by nohe427** [comment]
@Snugug - Can you take another look?

**Review by Snugug — APPROVED** [review]

**Comment by Snugug** [comment]
TYVM @nohe427! LGTG

**Review by Snugug — APPROVED** [review]

**Pull request merged**